test/setup.js

```diff
@@ -13,15 +12,4 @@ if (typeof global.document === 'undefined') {
 }
 
 global.chai = chai;
-
-_.forOwn(global.window, (value, key) => {
-  if (!window.hasOwnProperty(key)) {
-    return;
-  }
-
-  if (key in global) {
-    return;
-  }
-
-  global[key] = value;
-});
+global.navigator = window.navigator;
```

Comment:
This is the gist of the PR. We don't need to merge all of the properties of global.window into global. We just need the Navigator for React to render properly.